CHANGELOG.md

```diff
@@ -1,3 +1,5 @@
+* 0.6.3 - 2015-10-29
```

Comment:
This shouldn't specify a particular version or release date in the PR. I usually just write `HEAD` for the version number to track unreleased changes.